src/OV/System.cpp

```diff
@@ -113,6 +113,13 @@ OpenvarioSetRotation(DisplayOrientation orientation)
   WriteConfigFile(map, Path("/boot/config.uEnv"));
 }
 
+void
+OpenvarioSetLanguage(const char* lang)
+{
+  Run("localectl", "set-locale", lang);
```

Comment:
Did you test this? Does this work?

Reply:
No, not tested yet. I should've marked that commit as WIP as well. Was just an idea as a first shot without further looking into it. I can keep such things out of this Draft PR if it wastes your time looking at it.